app/actors/curation_concerns/actors/add_as_member_of_collections_actor.rb

```diff
@@ -16,7 +16,10 @@ def update(attributes)
         # Maps from collection ids to collection objects
         def add_to_collections(collection_ids)
           return true unless collection_ids
+          # grab/save collections this user has no edit access to
+          other_collections = curation_concern.member_of_collections.select { |coll| !coll.edit_users.include?(user.email) }
```

Comment:
instead of `email` can you use `user_key`? Otherwise this will break institutions like PSU who use `username` as the primary key.